src/Api/Controller/UpdateUserController.php

```diff
@@ -62,7 +62,7 @@ protected function data(ServerRequestInterface $request, Document $document)
             $password = Arr::get($request->getParsedBody(), 'meta.password');
 
             if (! $actor->checkPassword($password)) {
-                throw new PermissionDeniedException;
+                throw new NotAuthenticatedException;
```

Comment:
This should not be in this PR?

Reply:
Ack I thought I put that into a different branch...